README.md

```diff
@@ -107,21 +107,21 @@ operating system will do the following:
 When you execute a shim, pyenv determines which Python version to use by
 reading it from the following sources, in this order:
 
-1. The `PYENV_VERSION` environment variable (if specified). You can use
-   the [`pyenv shell`](https://github.com/pyenv/pyenv/blob/master/COMMANDS.md#pyenv-shell) command to set this environment
-   variable in your current shell session.
+1. The `PYENV_VERSION` environment variable (if specified). You can use the
+   [`pyenv shell <version>`](https://github.com/pyenv/pyenv/blob/master/COMMANDS.md#pyenv-shell)
```

Comment:
The suggested formulation gives a wrong idea that the command can only be used in this way. (While there are actually two more.)

When referring to a command as a whole, they don't replicate its invocation syntax -- because that'd be unreadable and duplication.

This could be good if the text was formulated so that the code is presented not as representation of the entire command but as a _one possible invocation of it_ ("with `pyenv shell <version>`" instead of "with the `pyenv shell <version>` command"). But since the version can be selected with this command in other ways, too, claiming that this can only be done with this invocation is again misinformation.

(same goes for the other edits below)